launcher/launcher.go

```diff
@@ -239,6 +242,9 @@ func (l *launcher) reconfigureWait(timeout time.Duration) (*config.C, error) {
 				err := l.validator.Validate(update)
 				if err != nil {
 					l.log.Errorf("Config update validation failed: %v", err)
+					if errors.Is(err, cloudbeat_config.ErrBenchmarkNotSupported) {
```

Comment:
The launcher was designed in a way it is agnostic to cloudbeat or any `Beater` implementation.
In that case, we can extend the `Validator` to support a special kind of error that will send that degraded message.
The only difference will be that the `Validator` accepts `config.C` instead of our struct.

Reply:
In that case, we can update the validator behavior to return a boolean whether the configuration is valid or not.
So the validator will also receive the `management.Manager` to update the status.

I am not sure it's better as we still have a coupling between the launcher and the validator.